electrical/wiring/README.md

```diff
@@ -72,6 +74,8 @@ On the other side of the cable, you will attach 4 Dupont female crimps. You now
 
 Now we'll insert each crimped cable into the appropriate connectors. Start by writing the pinout on each of the connectors for easy troubleshooting using a sharpie: B, A, G, 5 for channel B, channel A, ground, and +5V respectively. Copy over the annotations from the images below to your connectors, paying close attention to the orientation of each connector.
 
+![Encoder Extension Harness Diagram](images/encoder_extension.jpg)
```

Comment:
this is the first wiring harness encountered, a link or sentence or two about what to look for in this harness would be useful.

The DuPont connector displayed in this image seems to have 10 positions while it should only have 4. Is that an easy fix?

Reply:
I did worry about the DuPont connector image. Unfortunately, I pulled that image from the datasheet, which just uses the 10-position connector as a default, and there doesn't seem to be a reference for just the 4-position. Would it be less confusing if I removed the image? After all, a DuPont connector isn't as direction-sensitive as a Molex or a JST connector.

Reply:
Either that or use a cropped version of something like [the image here](https://www.dfrobot.com/product-2554.html), which also happens to be a really nice wiring diagram. Do you happen to recognize what software they might use for this?

Reply:
That is quite nice, I wish I knew what software they're using. It almost seems like it could also be a GraphViz derivative like WireViz, but I'm not sure.

I'm currently working on a fix to the JST and Molex connector diagrams, I might be able to do the DuPont as well while I'm at it.

Reply:
Commit db7e185